source/opt/common_uniform_elim_pass.h

```diff
@@ -178,9 +178,6 @@ class CommonUniformElimPass : public Pass {
   // Def-Uses for the module we are processing
   std::unique_ptr<analysis::DefUseManager> def_use_mgr_;
 
-  // Map from function's result id to function
```

Comment:
Thanks for deleting these now-unused variables.  That kind of cleanup is easy to miss. :-)